conf/locale/es_419/LC_MESSAGES/django.po

```diff
@@ -19018,10 +19016,8 @@ msgid ""
 "When you are ready to generate certificates for your course, click Generate "
 "Certificates. You do not need to do this if you have set the certificate "
 "mode to on-demand generation."
-msgstr ""
-"Cuando esté listo para generar los certificados de su curso, haga clic en "
-"Generar Certificados. No tendrá que hacer esto si ha seleccionado el modo de"
-" certificados por demanda."
+msgstr "When you are ready to generate certificates for your course, click Generate Certificates. You do not need to do this if you ave set the certificate mode to on-demand generation."
```

Comment:
make all long lines according to PEP requirements
line above is very good example how to do this

Reply:
@sendr 
Done